docs/en/developers/01_First_steps.md

```diff
@@ -1,6 +1,47 @@
-# Environment configuration
+# Environment configuration (Docker)
 
-**TODO**
+FreshRSS is built with PHP and use a homemade framework, Minz. The dependencies are directly included in the source code and so you don't need Composer.
+
+There are various ways to configure your development environment, the easiest and most supported one is based on Docker. This is the solution documented below. If you already have a working PHP environment, you probably don't need it.
+
+We assume here that you use a GNU/Linux distribution, capable to run Docker. Otherwise, you'll have to adapt the commands to your case.
+
+The commands that follow have to be executed in a console. They start by `$` when commands need to be executed as normal user, and by `#` when they need to be executed as root user. You don't have to type these characters. A path may be indicated before to help you to identify where they need to be executed. For instance, `app$ echo 'Hello World'` indicates that you have to execute `echo` command under `app/` directory.
+
+First, you need to install [Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
+
+Once you're done, clone the repository with:
+
+```console
```

Comment:
I wasn't aware GH had this one. Clever.

Reply:
I discovered that on Gitlab, I didn't even check that it was working on GitHub :sweat_smile:

Reply:
It seems to work fine https://github.com/flusio/FreshRSS/blob/tec/improve-development-environment/docs/en/developers/01_First_steps.md